public/javascripts/SVLabel/src/SVLabel/task/Task.js

```diff
@@ -35,6 +35,7 @@ function Task (geojson, currentLat, currentLng) {
 
         self.setProperty("streetEdgeId", _geojson.features[0].properties.street_edge_id);
         self.setProperty("completionCount", _geojson.features[0].properties.completion_count);
+        self.setProperty("priority", _geojson.features[0].properties.priority);
```

Comment:
Since NewTask objects now have priority with them, I store the priority in the Task objects on the front-end (getters below as well).